app/services/email_service/api/addresses.rb

```diff
@@ -29,17 +36,27 @@ def create(community_id:, opts:)
 
     private
 
-    def with_smtp_format(address)
-      address.merge(formatted: to_smtp_format(name: address[:name], email: address[:email]))
+    def with_formatted(address)
+      address.merge(
+        formatted: to_format(name: address[:name], email: address[:email], quotes: false),
```

Comment:
Suggestion to change names to indicate the intended use case better:
formatted -> display_format
smtp_formatted -> smtp_format